### DISCUSSION TITLE
New article on quantifying the effects of urban and agricultural trees to national carbon stocks

### DISCUSSION THREAD

**Discussion by advythr** [earthlab-education/Earth-Analytics-2023-01-Intro#288] [Awesome science]
[The overlooked contribution of trees outside forests to tree cover and woody biomass across Europe](https://www.science.org/doi/10.1126/sciadv.adh4097#).

Super cool article estimated carbon stocks of trees using forest inventory data and used remote sensing to quantify tree carbon stocks across Europe. Really shows how remote sensing can be used to answer questions at huge scales!

**Comment by nbol48** [comment]
This is really a cool application, and has the potential to open the carbon market to a large number of ag/municipal participants.

**Comment by eculler** [comment]
You can also check out friend of Earthlab's Max Joseph's [talk at our seminar on carbon accounting in the Southeastern US](https://earthlab.colorado.edu/blog/integrating-remote-sensing-data-forest-carbon-accounting)
